logging/tests/unit/test_logger.py

```diff
@@ -155,7 +155,30 @@ def test_log_text_w_timestamp(self):
         self.assertEqual(api._write_entries_called_with,
                          (ENTRIES, None, None, None))
 
-    def test_log_text_w_unicode_explicit_client_labels_severity_httpreq(self):
+    def test_log_text_w_trace(self):
+
+        TEXT = 'TEXT'
+        TRACE = '12345678-1234-5678-1234-567812345678'
+        ENTRIES = [{
+            'logName': 'projects/%s/logs/%s' % (
+                self.PROJECT, self.LOGGER_NAME),
+            'textPayload': TEXT,
+            'resource': {
+                'type': 'global',
+                'labels': {},
+            },
+            'trace': TRACE
+        }]
+        client = _Client(self.PROJECT)
+        api = client.logging_api = _DummyLoggingAPI()
+        logger = self._make_one(self.LOGGER_NAME, client=client)
+
+        logger.log_text(TEXT, trace=TRACE)
+
+        self.assertEqual(api._write_entries_called_with,
+                         (ENTRIES, None, None, None))
+
+    def test_log_text_w_unicode_explicit_client_labels_severity_httpreq_trace(self):
```

Comment:
lint error: line too long.

Given that you've added a separate testcase for `logger.log_text(...trace=...)`, we can probably just revert this one (the name, at a minimum).